.gitignore

```diff
@@ -1,5 +1,5 @@
 bin
 *.import
 .import
-*.translation
-.godot
\ No newline at end of file
+game/translations/*.translation
```

Comment:
What's the point of this in the context of this PR? If it doesn't make sense, please remove the change.

Reply:
Done